src/test/java/duke/task/TodoTest.java

```diff
@@ -0,0 +1,17 @@
+package duke.task;  //same package as the class being tested
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TodoTest {
+    @Test
+    public void dummyTest(){
+        assertEquals(2, 2);
+    }
+
+    @Test
+    public void anotherDummyTest(){
+        assertEquals(4, 4);
+    }
+}
```

Comment:
Overall, I think you did a very good job with the coding standard. There were barely any violations to point out. Everything is well formatted, named and documented